web/src/lib/domain-logic/filterFundings.test.ts

```diff
@@ -292,12 +292,12 @@ describe("filterFundings", () => {
       const funding1 = generateFunding({
```

Comment:
[Pebble]: In `filterFundings.ts` we added logic to account for `undefined/null`. I believe this can be removed since the default value is an array -> i.e.`if (it.certifications !== undefined && it.certifications !== null) {`

Reply:
Great memory and callout.